src/edu/csus/ecs/pc2/ui/team/CreateTeamsTSV.java

```diff
@@ -0,0 +1,268 @@
+        ParseArguments arguments = new ParseArguments(args, argsWithValues);
+
+        if (arguments.isOptPresent("--help")) {
+            usage();
+            System.exit(4);
+        }
+
+        debugMode = arguments.isOptPresent("--debug");
+        
+        
+        allowOverWrite = arguments.isOptPresent(OVERWRITE_FILE_KEY);
+        
+
+        if (debugMode) {
+            arguments.dumpArgs(System.err);
+        }
+
+        if (arguments.isOptPresent(OUTPUT_FILE_KEY)) {
+            outputFileName = arguments.getOptValue(OUTPUT_FILE_KEY);
+        }
+        
+        if (arguments.isOptPresent(INPUT_FILE_KEY)) {
+            inputFileName = arguments.getOptValue(INPUT_FILE_KEY);
+        }
+        
+
+    }
+
+    private void fatalError(String errorMessage) {
+        System.err.println("Fatal error " + errorMessage);
+        System.exit(FAILURE_EXIT_CODE);
+
+    }
+
+    private static void usage() {
+        String[] usage = { //
+                "", //
+                "Usage CreateTeamsTSV [--help] [-f] [--of outputfilename] [--if teams_file.json]", //
+                "", //
+                "Purpose: read teams.json file create teams.tsv and groups.tsv files.", //
+                "", //
+                "This will not overwrite output file, use -f to overwrite output file (and "+GROUPS_TSV_FILENAME+")", //
+                "", //
+                "options:", //
+                "--of file  - alternate output filename, default output filename is " + TEAMS_TSV_FILENAME, //
+                "--if file  - alternate input JSON filename, default  filename is " + TEAMS_JSON_FILENAME, //
+                     "-f   overwrite output file, by default will not overwrite output file.", //
+                "", //
+                "On success exit code will be " + SUCCESS_EXIT_CODE, //
+                "Any other exit code is an error.", //
+                "", //
+        };
+
+        for (String s : usage) {
+            System.out.println(s);
+        }
+    }
+
+    protected void writeTSVFile() throws Exception {
+
+        if (!allowOverWrite  && new File(outputFileName).exists()) {
+            fatalError("Output file exists (" + outputFileName + "), will not overwrite.  Use " + OUTPUT_FILE_KEY + " to write to a different file");
+        }
+
+        List<TeamAccount> accounts = loadAccounts(inputFileName);
+
+        String [] header = { "Teams", "1" };
+
+        String headerLine = String.join(TAB, header);
+
+        FileWriter writer = (new FileWriter(new File(outputFileName)));
+
+        writer.write(headerLine + NL);
+
+        for (TeamAccount teamAccount : accounts) {
+            // TODO write teams.tsv output file
+            //            System.out.println(" debug 22 " + teamAccount.toJSON());
+
+            String groupId = "1";
+            if (teamAccount.getGroup_ids().size() > 0) {
+                groupId = teamAccount.getGroup_ids().get(0);
```

Comment:
I assume you're calling `get(0)` because PC2 can't handle more than one group for a given team account?
If so, it might be nice to point that out in a comment (or even a TODO)...